opennms-base-assembly/src/main/filtered/etc/datacollection/ref_mib2-host-resources.xml

```diff
@@ -0,0 +1,60 @@
+      <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
+   </resourceType>
+   <resourceType name="hrDeviceEntry" label="Host Resources Device" resourceLabel="${hrDeviceDescr} (${index})">
+      <persistenceSelectorStrategy class="org.opennms.netmgt.collectd.PersistRegexSelectorStrategy">
+         <parameter key="match-expression" value="(#hrDeviceType == '.1.3.6.1.2.1.25.3.1.3')"/>
+      </persistenceSelectorStrategy>
+      <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
+   </resourceType>
+   <group name="mib2-host-resources-storage" ifType="all">
+      <mibObj oid=".1.3.6.1.2.1.25.2.3.1.2" instance="hrStorageIndex" alias="hrStorageType" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.25.2.3.1.3" instance="hrStorageIndex" alias="hrStorageDescr" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.25.2.3.1.4" instance="hrStorageIndex" alias="hrStorageAllocUnits" type="gauge"/>
+      <mibObj oid=".1.3.6.1.2.1.25.2.3.1.5" instance="hrStorageIndex" alias="hrStorageSize" type="gauge"/>
+      <mibObj oid=".1.3.6.1.2.1.25.2.3.1.6" instance="hrStorageIndex" alias="hrStorageUsed" type="gauge"/>
+   </group>
+   <group name="mib2-host-resources-storage-non-unique" ifType="all">
+      <mibObj oid=".1.3.6.1.2.1.25.2.3.1.2" instance="hrStorageNonUnique" alias="hrStorageType" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.25.2.3.1.3" instance="hrStorageNonUnique" alias="hrStorageDescr" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.25.2.3.1.4" instance="hrStorageNonUnique" alias="hrStorageAllocUnits" type="gauge"/>
+      <mibObj oid=".1.3.6.1.2.1.25.2.3.1.5" instance="hrStorageNonUnique" alias="hrStorageSize" type="gauge"/>
+      <mibObj oid=".1.3.6.1.2.1.25.2.3.1.6" instance="hrStorageNonUnique" alias="hrStorageUsed" type="gauge"/>
+   </group>
+   <group name="mib2-host-resources-system" ifType="ignore">
+      <mibObj oid=".1.3.6.1.2.1.25.1.1" instance="0" alias="hrSystemUptime" type="timeticks"/>
+      <mibObj oid=".1.3.6.1.2.1.25.1.5" instance="0" alias="hrSystemNumUsers" type="gauge"/>
+      <mibObj oid=".1.3.6.1.2.1.25.1.6" instance="0" alias="hrSystemProcesses" type="gauge"/>
+   </group>
+   <group name="mib2-host-resources-memory" ifType="ignore">
+      <mibObj oid=".1.3.6.1.2.1.25.2.2" instance="0" alias="hrMemorySize" type="integer"/>
+   </group>
+   <group name="mib2-host-resources-processor" ifType="all">
+      <mibObj oid=".1.3.6.1.2.1.25.3.2.1.1" instance="hrDeviceEntry" alias="hrDeviceIndex" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.25.3.2.1.2" instance="hrDeviceEntry" alias="hrDeviceType" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.25.3.2.1.3" instance="hrDeviceEntry" alias="hrDeviceDescr" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.25.3.3.1.2" instance="hrDeviceEntry" alias="hrProcessorLoad" type="Gauge32"/>
+   </group>
+   <systemDef name="Enterprise MIB2-Host-Resources">
+      <sysoidMask>.1.3.6.1.4.1.</sysoidMask>
```

Comment:
I don't think the above is relevant for network devices. HOST-RESOURCES-MIB is relevant for certain (not necessarily all) SNMP agents you can find on server operating systems only.